frontend/src/bundles/common/components/transanction-table/components/transaction/transaction.tsx

```diff
@@ -6,21 +8,21 @@ import {
     type FieldValues,
 } from 'react-hook-form';
 
-import OrangeMurseIcon from '~/assets/img/orange-murse-icon.svg';
 import { Input } from '~/bundles/common/components/components';
 import { getDaysLeft } from '~/bundles/common/components/transanction-table/helpers/index';
 import { InputType } from '~/bundles/common/enums/enums';
 
 import { type ITransaction } from '../../types';
 import styles from '../styles.module.scss';
 
-interface TransactionProperties {
+interface Properties {
```

Comment:
we are using type in our project

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/367/commits/21ca00bfc63bfbb4356b47d30687f448c8225403](url)